assignments/resume-bootstrap/Resume-v2.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Mohamed Shahin</title>
+    <link rel="stylesheet" href="bootstrap.min.css" />
+    <style>
+        header {
+            margin-bottom: 5px;
+            border-bottom: 2px solid black;
+            padding-bottom: 10px;
+            overflow: hidden;
+        }
+        
+        header h1 {
+            font-size: 28px;
+            font-family: 'Times New Roman', Times, serif;
+        }
+        
+        header h2 {
+            font-size: 22px;
+            font-family: 'Times New Roman', Times, serif;
+        }
+        
+        header p {
+            font-size: 13px;
+        }
+        
+        header h3 {
+            font-size: 15px;
+        }
+        
+        header img {
+            max-height: 180px;
+            max-width: 180px;
+            box-sizing: border-box;
+        }
+        
+        @media (max-width :768) {
+            header img {
```

Comment:
You are already using hidden-sm and hidden-xs to remove the image on smaller screens so why do you need this styling?